### PR TITLE
Remove member area from menu (desktop and mobile), when user has no elopage account

### LINKED ISSUE #2094: 🔧 [Refactor] Remove Member Area from Menu
## 🔧 Refactor ticket

The link to the elopage member area is no longer needed for users without elopage. Please remove, if user has no elopage.

Remove the link to the members area from the footer for all users.

### PR DESCRIPTION
**Pull request by mahula**
## 🍰 Pullrequest
The meber area entry is not displayed in the footer and not in the menu, if the user has no elopage account.

### Issues
- fixes #2094

### Todo
- [X] None

### CODE REVIEW

File: frontend/src/components/Menu/Sidebar.spec.js [hunk 1]
```diff
@@ -55,7 +51,12 @@ describe('Sidebar', () => {
       it('has first nav-item "navigation.profile" in navbar', () => {
         expect(wrapper.findAll('.nav-item').at(4).text()).toEqual('navigation.profile')
       })
-
+    })
+    // ----
+    describe('navigation Navbar (user has an elopage account)', () => {
+      it('has seven b-nav-item in the navbar', () => {
```
Comment (Mogge) [inline frontend/src/components/Menu/Sidebar.spec.js:57]: eight?

File: frontend/src/components/Menu/Sidebar.spec.js [hunk 2]
```diff
@@ -69,5 +70,20 @@ describe('Sidebar', () => {
         expect(wrapper.findAll('.nav-item').at(7).text()).toEqual('navigation.logout')
       })
     })
+    describe('navigation Navbar (user has no elopage account)', () => {
+      beforeAll(() => {
+        mocks.$store.state.hasElopage = false
+        wrapper = Wrapper()
+      })
+      it('has six b-nav-item in the navbar', () => {
```
Comment (Mogge) [inline frontend/src/components/Menu/Sidebar.spec.js:78]: seven?

**Review by Mogge — CHANGES_REQUESTED** [review]
Looks very good. Some small things that may have been wrong before.
We like to have a blank line before each it and describe block.

**Comment by mahula** [comment]
> We like to have a blank line before each it and describe block.

Sure. This has been fixed with [bb120b8](https://github.com/gradido/gradido/pull/2099/commits/bb120b8b82f9c25f5caa39cdd81064c1a36e4dc9).

File: frontend/src/components/Menu/Sidebar.spec.js (hunk 1 above)
Comment (mahula) [reply frontend/src/components/Menu/Sidebar.spec.js]: Das ist korrekt.
Das Wording wurde von vorheriger version des tests übernommen.
Aber es sind jeweils acht (user has elopage account) und sieben (user does not have elopage account) nav items.

Fixed with [a413851](https://github.com/gradido/gradido/pull/2099/commits/a41385141902b7611c7fbde289d68cc3d2fe4c6b).

File: frontend/src/components/Menu/Sidebar.spec.js (hunk 2 above)
Comment (mahula) [reply frontend/src/components/Menu/Sidebar.spec.js]: Same as above.

**Review by Mogge — APPROVED** [review]

**Pull request merged**